src/database/remove.ts

```diff
@@ -0,0 +1,98 @@
+import * as pathLib from "path";
+import { Queue } from "../queue";
+import * as FirebaseError from "../error";
+import * as logger from "../logger";
+import RemoveRemote, { PrefetchResult } from "./remove-remote";
+
+export interface DatabaseRemoveOptions {
+  // RTBD instance ID.
+  instance: string;
+  // Number of concurrent chunk deletes to allow.
+  concurrency: number;
```

Comment:
Are all of these required? Or should they default (in the consructor) if not specified?